src/mote/templates/teams.html

```diff
@@ -0,0 +1,380 @@
+                            23 Groups
+                        </div>
+                        <table class="table">
+                            <tbody>
+                                {% for i in teamdata %}
+                                <tr>
+                                    <td class="item">{{i}}</td>
+                                    <td class="item">
+                                        <item>
+                                            Latest Minutes:
+                                        </item>
+                                        <div class="btn-group" role="group"
+                                            aria-label="Basic checkbox toggle button group">
+                                            <input type="checkbox" class="btn-check" id="btncheck1" autocomplete="off">
+                                            <label class="btn btn-outline-primary" for="btncheck1"><i
+                                                    class="far fa-calendar-alt"></i> 15 Nov 2021</label>
+                                        </div>
+                                    </td>
+                                    <td style="text-align: right;">
+                                        <div class="btn-group" role="group" aria-label="Basic outlined example">
+                                            <button type="button" class="btn btn-outline-primary"><i
+                                                    class="fas fa-clock"></i>
+                                                4</button>
+                                            <button type="button" class="btn btn-outline-primary"> <i
+                                                    class="fas fa-archive"></i> 124</button>
+                                        </div>
+                                    </td>
+                                </tr>
+                                {% endfor %}
+                            </tbody>
+                        </table>
+                    </div>
+                </div>
+            </div>
+            <div class="card-footer text-center" style="background-color: #e6e6e6; color:  #676767;">
+                <span class="small">
+                    <a style="color:#676767; text-decoration:none;" href="https://github.com/t0xic0der/fragment/"
+                        target="_blank">
+                        Fragment
+
+                    </a>
+                    &copy 2021
+
+                    <a style="color:#676767; text-decoration:none;" href="https://github.com/t0xic0der" target="_blank">
+                        Akashdeep Dhar
+                    </a>
+                    and
+                    <a style="color:#676767; text-decoration:none;"
+                        href="https://github.com/t0xic0der/fragment/graphs/contributors" target="_blank">
+                        others
+                    </a>
+                </span>
+            </div>
+        </div>
+    </div>
+</body>
+<script>
```

Comment:
Most of this is related to the calendar plugin, which is not on this page.
It should probably be removed.